packages/cli/src/callbacks.ts

```diff
@@ -187,12 +187,13 @@ export const getConfigWithHeader = async (output: WriteStream, credentialsType:
 }
 
 export const getEnvName = async (currentName = 'env1'): Promise<string> => {
-  const questions = [{
+  const questions: Array<inquirer.Question> = [{
```

Comment:
nit
```suggestion
  const questions: inquirer.Questions = [{
```
I guess both work though since `inquirer.Questions` can be `Array<inquirer.Question>`